api/tests/unit/application/authentication/oidc/oidc-controller_test.js

```diff
@@ -492,4 +493,31 @@ describe('Unit | Application | Controller | Authentication | OIDC', function ()
       expect(result.source).to.deep.equal({ access_token: 'accessToken', logout_url_uuid: 'logoutUrlUUID' });
     });
   });
+
+  describe('#reconcileUserForAdmin', function () {
+    it('should call use case and return the result', async function () {
```

Comment:
```suggestion
    it('calls use case and return the result', async function () {
```

Reply:
les should ne me dérangent pas mais je crois que c'est dans vos pratiques accès ^^
Je les corrigerai si j'ai le temps mais je priorise les autres retours